email_test.go

```diff
@@ -717,13 +717,19 @@ TGV0J3MganVzdCBwcmV0ZW5kIHRoaXMgaXMgcmF3IEpQRUcgZGF0YS4=
 	if e.Attachments[0].Filename != a.Filename {
 		t.Fatalf("Incorrect attachment filename %s != %s", e.Attachments[0].Filename, a.Filename)
 	}
-	if !bytes.Equal(e.Attachments[0].Content, a.Content) {
+	var b1, b2 []byte
+	b1, _ = streamToBytes(e.Attachments[0].Content)
```

Comment:
Wait, here we're still storing the full content of the attachment into memory, no? After noticing this, I don't actually se where are we saving memory wit this PR. Do we have measurement of memory use to confirm if we got any improvement?